src/test/resources/application-test.yml

```
@@ -19,7 +19,49 @@ spring:
+              - account_email
+        provider:
+          kakao:
+            authorization-uri: https://kauth.kakao.com/oauth/authorize
+            token-uri: https://kauth.kakao.com/oauth/token
+            user-info-uri: https://kapi.kakao.com/v2/user/me
+            user-name-attribute: id
+
+---
+jwt:
+  secret: abcdefghijklmnopqrstuvwxyz123456abcdefghijklmnopqrstuvwxyz123456abcdefghijklmnopqrstuvwxyz123456abcdefghijklmnopqrstuvwxyz123456
+  access-expiration: 2592000000 # 30일
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)